batch/batches/StorageUpdate/KAsyncStorageUpdate.php

```diff
@@ -238,7 +267,14 @@ public function handleUsage($percent, $partner, $systemPartnerConfiguration, $mo
 				}
 				else
 				{
-					$this->systemPartnerClientPlugin->systemPartner->updateStatus($partner->id, KAsyncStorageUpdateUtils::PARTNER_STATUS_CONTENT_BLOCK, $reason);
+					try
+					{

```

Comment:
To reduce repeating code - 
Add small function partnerUpdateStatus(status) that will do all the try catch internally